package.json

```
@@ -180,470 +180,78 @@
     ]
   },
   "typings": "./handsontable.d.ts",
+  "handsontable": {
+    "copy": [
+      "dist/handsontable.css",
+      "dist/handsontable.full.css",
+      "dist/handsontable.full.js",
+      "dist/handsontable.full.min.css",
+      "dist/handsontable.full.min.js",
+      "dist/handsontable.js",
+      "dist/handsontable.min.css",
+      "dist/handsontable.min.js",
+      "dist/languages",
+      "dist/README.md",
+      "languages",
+      "base.d.ts",
+      "CHANGELOG.md",
+      "handsontable-general-terms.pdf",
+      "handsontable-non-commercial-license.pdf",
+      "handsontable.d.ts",
+      "LICENSE.txt",
+      "README.md"
+    ],
+    "exports": [
+      "./cellTypes/**/+(index|registry)*",
+      "./languages/!(all.js)",
+      "./editors/**/+(index|registry)*",
+      "./plugins/**/+(index|registry)*",
+      "./renderers/**/+(index|registry)*",
+      "./validators/**/+(index|registry)*",
```

```suggestion
      "./cellTypes/*/+(index)*",
      "./cellTypes/+(index|registry)*",
      "./languages/!(all.js)",
      "./editors/*/+(index)*",
      "./editors/+(index|registry)*",
      "./plugins/*/+(index)*",
      "./plugins/+(index|registry)*",
      "./renderers/*/+(index)*",
      "./renderers/+(index|registry)*",
      "./validators/*/+(index)*",
      "./validators/+(index|registry)*",
```
Ups.. the current patterns search the files too deeply. For example, the register for column sorting service, in that case, is exported. It matches to the `./plugins/columnSorting/sortService/registry.mjs` file. 

My suggestion splits the glob patterns. The first pattern `"./plugins/*/+(index)*"` searches index (js and mjs) files only one level in found directories. And the second search only the index and registry files within the main directory. 

If you have a clue how to write the pattern that includes both cases feel free to adjust the above suggestion.